### PR TITLE
Version update v4.5.7

### PR DESCRIPTION
**Pull request by dharmin**
### 4.5.7 [June 12, 2019] ###

* NEW
    * Add new filter to remove the media upload button from the activity [Doc](https://rtmedia.io/docs/developers/hooks/rtmedia-enable-activity-media-upload/)
    * Add new filter to change the order of the media and text on the activity page [Doc](https://rtmedia.io/docs/developers/hooks/rtmedia-activity-content-html/)

* FIXED
    * Duplicate media comment issue on Activity page
    * Add title attribute for Media Likes
    * Improve UI of Media Page title
    * Issue of entering text in text area when Direct Upload and Upload terms enabled together
    * UI of pagination button height and text field
    * Export/Import button issue in Firefox browser
    * Issue of direct upload and media was not opening in lightbox with TwentySeventeen theme v2.1 
    * Empty space issue for generated activity when media is uploaded without adding text 
    * Other Errors, Warnings and notices
    * Group Activity media are not shown in Group Media tab with BuddyPress Nouveau Template
    * User’s name autosuggestion
    * Issue with multiple file upload from activity with non admin user account

### CODE REVIEW

**Comment by nitun** [comment]
@dharmin the date should be 11 June. Not 12 June.

**Comment by sanketio** [comment]
@thrijith @dharmin : Added couple of feedbacks, please check and add comments in reply. Thanks!

File: app/main/controllers/template/rtmedia-functions.php [hunk 1]
```diff
@@ -2880,7 +2886,7 @@ function show_rtmedia_like_counts() {
 		}
 		?>
 		<div class='rtmedia-like-info <?php echo $class; ?>'>
-			<i class="rtmicon-thumbs-up rtmicon-fw"></i>
+			<i></i>
```
[the remark this answers is not in the document]
Comment (dharmin) [reply app/main/controllers/template/rtmedia-functions.php]: We might need this for the like icon.

[the remark this answers is not in the document]
Comment (sanketio) [reply app/main/controllers/template/rtmedia-functions.php]: Did you check by removing it?

File: app/main/controllers/activity/RTMediaActivity.php [hunk 2]
```diff
@@ -53,57 +68,122 @@ function create_activity_html( $type = 'activity' ) {
...
+		 * @param string $media_list       HTML markup of media in ul.
+		 */
+		$activity_content = apply_filters( 'rtmedia_activity_content_html', $activity_text . $media_list, $activity_text, $media_list );
+
+		$activity  = $activity_container_start;
+		$activity .= $activity_content;
+		$activity .= $activity_container_end;
+
+		// Bypass comment links limit.
+		add_filter(
```
[the remark this answers is not in the document]
Comment (dharmin) [reply app/main/controllers/activity/RTMediaActivity.php]: We're don't need to remove the filter, So, I've added the anonymous function.

**Pull request merged**